Assets/Dialogue/Barks/BrutusOfficeBarks/BrutusOfficeIdleBark 3.asset

```diff
@@ -12,4 +12,4 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 3a4c60ef165185e4d9c0e38f4bee7913, type: 3}
   m_Name: BrutusOfficeIdleBark 3
   m_EditorClassIdentifier: 
-  line: Maybe I should stop eating so my mom doesn't call me fat
+  line: Maybe I should stop eating so much so my mom doesn't call me fat
```

Comment:
Maybe instead of "fat" we can use some like funny nickname relating to him being fat?
Like "chubby cakes" or something LOL

Reply:
Good idea!